src/dotnet/commands/dotnet-migrate/Program.cs

```diff
@@ -21,29 +21,50 @@ public static int Run(string[] args)
 
             CommandLineApplication app = new CommandLineApplication();
             app.Name = "dotnet migrate";
-            app.FullName = ".NET Migrate Command";
-            app.Description = "Command used to migrate project.json projects to msbuild";
+            app.FullName = LocalizableStrings.AppFullName;
+            app.Description = LocalizableStrings.AppDescription;
             app.HandleResponseFiles = true;
             app.HelpOption("-h|--help");
 
-            CommandArgument projectArgument = app.Argument("<PROJECT_JSON/GLOBAL_JSON/PROJECT_DIR>",
+            CommandArgument projectArgument = app.Argument(
+                $"<{LocalizableStrings.CmdProjectArgument}>",
                 string.Join(Environment.NewLine,
-                "The path to ",
-                " - a project.json file to migrate.",
-                "or",
-                " - a global.json file, it will migrate the folders specified in global.json.",
-                "or",
-                " - a directory to migrate, it will recursively search for project.json files to migrate.",
-                "Defaults to current directory if nothing is specified."));
+                LocalizableStrings.CmdProjectPath,
+                LocalizableStrings.CmdProjectJson,
+                LocalizableStrings.CmdOr,
+                LocalizableStrings.CmdProjectGlobalJson,
+                LocalizableStrings.CmdOr,
+                LocalizableStrings.CmdProjectDirectory,
+                LocalizableStrings.CmdProjectDirectoryDescription));
 
-            CommandOption template = app.Option("-t|--template-file", "Base MSBuild template to use for migrated app. The default is the project included in dotnet new.", CommandOptionType.SingleValue);
-            CommandOption sdkVersion = app.Option("-v|--sdk-package-version", "The version of the sdk package that will be referenced in the migrated app. The default is the version of the sdk in dotnet new.", CommandOptionType.SingleValue);
-            CommandOption xprojFile = app.Option("-x|--xproj-file", "The path to the xproj file to use. Required when there is more than one xproj in a project directory.", CommandOptionType.SingleValue);
-            CommandOption skipProjectReferences = app.Option("-s|--skip-project-references", "Skip migrating project references. By default project references are migrated recursively.", CommandOptionType.BoolValue);
+            CommandOption template = app.Option(
+                "-t|--template-file",
+                LocalizableStrings.CmdTemplateDescription,
```

Comment:
Yes, I would prefer that. I would prefer for us to be as consistent as possible with ourselves. For instance, https://github.com/dotnet/cli/pull/4891/files#diff-4649f1d60eb7d9a20c352c411241e587R15, does not have cmd. Also, other LocalizableStrings already merged do not have the Cmd.